'''General helpers for working with exception handling'''


def _get_friendly_error(problem=str()):
    pass
